### PR TITLE
Added option to set price manually in bulk add

### PR DESCRIPTION
**Pull request by arik123**
image
image

#32

### CODE REVIEW

File: app/pricelist.js [hunk 1]
```diff
@@ -28,96 +28,107 @@ pricelist.addItems = async function(search, options) {
...
-		.then(async (prices) => {
-			console.log('Got all prices, continuing...');
-
-			for (let i = 0; i < prices.length; i++) {
-				if (search.indexOf(prices[i].name) > -1) {
-					skus.push(prices[i].sku);
-					search.splice(search.indexOf(prices[i].name), 1);
-				}
+	if (options.autoprice) {
+		const prices = await getAllPrices();
```
Comment (ZeusJunior) [inline app/pricelist.js:32]: Should maybe catch an error here

File: views/index.ejs [hunk 2]
```diff
@@ -159,6 +159,36 @@
 								<label for="url">Items</label>
 								<textarea rows="5" class="form-control" id="url" name="input" placeholder="Add the backpack.tf stats page, sku, or the items name. Seperate multiple inputs by a new line. See console for speed" required v-model="bulk.input"></textarea>
 							</div>
+							<div class="form-row">
+								<div class="col">
+									  <div class="pretty p-switch p-fill">
+										<input type="checkbox" name="autoprice" v-model="bulk.autoprice"/>
```
Comment (ZeusJunior) [inline views/index.ejs:165]: Should make autoprice ON the default

Comment (arik123) [reply views/index.ejs:165]: it is set in index.js

File: app/pricelist.js (hunk 1 above)
Comment (arik123) [reply app/pricelist.js:32]: Errors wil get caught in addItems.js, no need to rethrow them.

**Pull request merged**